### DISCUSSION TITLE
Built for Shopify

### DISCUSSION THREAD

**Discussion by zeeshanhaider6236dev** [Kyon147/laravel-shopify#155] [General]
Largest Contentful Paint tracks how quickly your app is able to display its main content to merchants. A good score is less than 2.5 seconds.

Anyone got badge ( buit for shopify )? i'm stuck in this LCP my load 4.1 

how can i detect and improve this score and apply for program anyone know more about this ?

**Comment by Kyon147** [comment]
@zeeshanhaider6236dev you can install web vitals in your app and track the values yourself manually and store them in a DB or off-site. 

LCP really depends on how your app is built, and how good your server/service is at serving your application.  I have a LCP on one of my apps at 1.5s so it is doable.
